### PR TITLE
fix multiple rubocop violations

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

File: lib/beaker-rspec/helpers/serverspec.rb [hunk 1]
```diff
@@ -150,7 +150,7 @@ def self.run(meth, *args)
       cmd = Specinfra.command.get(meth, *args)
       backend = Specinfra.backend
       ret = backend.run_command(cmd)
-      if meth.to_s =~ /^check/
+      if /^check/.match?(meth.to_s)
```
Comment (ekohl) [inline lib/beaker-rspec/helpers/serverspec.rb:153]: Any thoughts on:
```suggestion
      if meth.to_s.start_with('check')

File: lib/beaker-rspec/helpers/serverspec.rb [hunk 2]
```diff
@@ -80,7 +80,7 @@ def detect_os
     return Specinfra.configuration.os if Specinfra.configuration.os
     backend = Specinfra.backend
     node = get_working_node
-    if /windows/.match?(node['platform'])
+    if node['platform'].include?('windows')
```
Comment (ekohl) [inline lib/beaker-rspec/helpers/serverspec.rb:83]: This line was also changed in a previous commit. I think if you order this before f91615f4c52e7c2dc9430bcc3b3554eeb907b490 (rubocop: fix Performance/RegexpMatch) the you have fewer changes.

**Review by ekohl — COMMENTED** [review]
Looks ok. Some small changes, but either way I think this is fine.

File: lib/beaker-rspec/helpers/serverspec.rb (hunk 1 above)
Comment (bastelfreak) [reply lib/beaker-rspec/helpers/serverspec.rb:153]: ah I think that's even a tiny bit faster.

File: lib/beaker-rspec/helpers/serverspec.rb (hunk 2 above)
Comment (bastelfreak) [reply lib/beaker-rspec/helpers/serverspec.rb:83]: I updated the ordering

**Review by ekohl — APPROVED** [review]
Thanks!

**Pull request merged**